.rubocop.yml

```
@@ -138,7 +139,7 @@ Style/SlicingWithRange:
   Enabled: true
 
 Lint/MixedRegexpCaptureTypes:
-  Enabled: true
+  Enabled: false
```

Can you test if it still passes with this enabled?
```suggestion
  Enabled: true
```

---

@ekohl it is enabled on my previous changes https://github.com/theforeman/kafo/commit/80f52596a07bddaecb2f0233072baeb1b081056d but now it is asking me to replace the named captures with numbered captures to avoid mixing them and which I did previously and that didn't look good! And the previous change is not satisfying this cop now. Do you have any suggestions to fix this or resources to look into, i think i'm missing something here.

---

We missed a group:
https://github.com/theforeman/kafo/blob/a1b1b12d47f2e0dd26629452809c915e40a861b4/lib/kafo/kafo_configure.rb#L522

This needs to be:
```ruby
            if (output = line.match(/(?:.+\]): Starting to evaluate the resource(?: \((?<count>\d+) of (?<total>\d+)\))?/))
```

---

Thank @ekohl, updated the changes.